src/test/java/core/basesyntax/BalanceOperationTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.operations.BalanceOperation;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
```

Comment:
Add more test cases for your `OperationHandler` implementations. 
For example:
`updateStorage_fruitAmountIsNegative_isNotOk`